packages/web-components/src/components/chat/components/messages/src/messages.template.ts

```diff
@@ -49,9 +49,9 @@ export function messagesTemplate(customElementClass) {
       aria-labelledby="${clabsPrefix}--chat-messages-target-reader-label"
       class="${clabsPrefix}--chat-messages-container ${streamResponses
         ? clabsPrefix + '--chat-messages-container-streaming'
-        : ''} 
-
-    ${dockingEnabled ? clabsPrefix + '--chat-messages-container-docked' : ''}">
+        : ''} ${dockingEnabled
```

Comment:
This is a little hard to follow with all the nested ternary's here - is there a better place I'm wondering... maybe before the return you could have an array and for each of those evaluated class names, if supported ( streamResponses, dockingEnabled... etc - is truthy ) push the class name... then your class assignment here you could just Array.split(" ").